apps/graphql/src/dataloaders/Itinerariesloader.js

```diff
@@ -82,20 +118,43 @@ const fetchItineraries = async (
       );
     }),
   );
-  return results.map(res => sanitizeIteneraries(res.data));
+  return results.map(res => {
+    return sanitizeItineraries(res);
+  });
 };
 
-const sanitizeIteneraries = (
-  itineraries: $PropertyType<ApiResponse, 'data'>,
-): Itineraries[] => {
+const sanitizeItineraries = (response: ApiResponse): Itineraries[] => {
+  const itineraries = response.data;
   return itineraries.map(itinerary => ({
     id: itinerary.id,
     airlines: itinerary.airlines,
-    price: itinerary.price,
+    price: {
+      currency: response.currency,
+      amount: itinerary.price,
+    },
     flyFrom: itinerary.flyFrom,
     flyTo: itinerary.flyTo,
     localDeparture: itinerary.local_departure,
     localArrival: itinerary.local_arrival,
+    routes: itinerary.routes,
+    route:
+      itinerary.route &&
+      itinerary.route.map(routeItem => ({
+        airline: routeItem.airline ?? '',
+        arrival: {
+          city: routeItem.cityTo ?? '',
+          cityCode: routeItem.flyTo ?? '',
+          localTime: routeItem.local_arrival ?? new Date(),
```

Comment:
I am not sure `new Date()` is a good default value. Maybe `null` instead?

Reply:
OK